W-18564680 mcp schema validation policy mb (#861)
mulesoft/docs-gateway: gateway/1.9/modules/ROOT/pages/policies-included-mcp-schema-validation.adoc

The MCP Schema Validation policy validates that every every JSON RPC request to the MCP server complies with the MCP schema (MCP 2024_11_05 standard). If validation fails it will report a Policy Violation.

There are no configuration parameters for this policy.

include::partial$policy-title-headers.adoc[tag=ui]

> The MCP Schema Validation policy is not supported in Local Mode.

Reviewer: ```suggestion
The MCP Schema Validation policy isn't supported in Local Mode.
```
